packages/web3-eth/test/unit/rpc_method_wrappers/fixtures/sign_transaction.ts

```diff
@@ -17,27 +17,20 @@ along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 import { Transaction } from 'web3-types';
 
 const transaction: Transaction = {
-	from: '0xEB014f8c8B418Db6b45774c326A0E64C78914dC0',
-	to: '0x3535353535353535353535353535353535353535',
-	value: '0x174876e800',
-	gas: '0x5208',
-	gasPrice: '0x4a817c800',
 	type: '0x0',
```

Comment:
We should also add tests for 0x1 and 0x2 transactions

Reply:
Updated via [this](https://github.com/ChainSafe/web3.js/pull/5312/commits/ec538fdc8b4e3844c391ec3ebe18d388a9bd497f) commit